### ISSUE TITLE
Can we please get the old Arcane Lamp behavior back?

### ISSUE THREAD

**Issue by AnrDaemon** [Azanor/thaumcraft-suggestions#249]
I could use painted glowstone to illuminate the tree, of course, but arcane lamps are just that much easier to manage.

**Comment by Azanor** [comment]
What is the old behaviour? The lamps jave worked pretty much the same for several versions now

**Comment by AnrDaemon** [comment]
Previously, the lamp was spawning glimmers regardless the line of sight, illuminating entire 32x32x32 area.
Now, it only lights the room it is placed in.

**Comment by Shahelion** [comment]
Looks like the area of effect has been reduced to 16x16, according to the Thaumonomicon, which is barely more than vanilla torches.

**Comment by AnrDaemon** [comment]
It is still "16 range", which amounts for (approximately) 32x32x32 block of "safe light level". Plus whatever adds on top of it.
But if you place the lamp in a closed room, the results are different. Very different.

**Comment by jvcmarcenes** [comment]
Yeah, I placed the lamp in a wall once, just a wall in the mid of nowhere really, not even inside a room, And the lights just appeared on the side that the lamp was on, seems like the lamp can't illuminate  blocks that it can't seem which does make sense
